app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SixLineAddressController.scala

```diff
@@ -129,11 +129,19 @@ class SixLineAddressController @Inject() (
 
   private def formsByOrganisationTypes(implicit request: Request[AnyContent]) = {
     val form = requestSessionData.selectedUserLocationWithIslands(request) match {
-      case Some("islands") => islandsSixLineAddressForm
-      case _               => thirdCountrySixLineAddressForm
+      case Some("islands")     => islandsSixLineAddressForm
+      case Some("isle-of-man") => iomSixLineAddressForm
```

Comment:
We have the UserLocation class which includes constants for these fields. Lets clean up the use of these magic strings and use those existing constants.